frontend/app/Components/BlogIndexPage/BlogIndexPage.js

```diff
@@ -4,16 +4,16 @@ import Categories from './Categories';
 
 export default function BlogIndexPage() {
   return (
-    <div className="bg-light_gray px-36 py-20">
-      <h1 className="text-6xl text-black">
+    <main className="bg-light_gray px-36 py-20">
+      <h2 className="text-6xl text-black">
```

Comment:
I’ve made a change here – there is only ever one `main` landmark on the page, so no need for it to be labelled.